u2f-ref-code/java/tests/com/google/u2f/TestVectors.java

```diff
@@ -291,6 +291,35 @@
   protected static final X509Certificate TRUSTED_CERTIFICATE_MULTIPLE_TRANSPORTS =
       parseCertificateBase64(TRUSTED_CERTIFICATE_MULTIPLE_TRANSPORTS_BASE64);
 
+  // Has Bluetooth Radio, Bluetooth Low Energy, NFC, and USB Internal
+  private static final String TRUSTED_CERTIFICATE_EXTENDED_TRANSPORTS_BASE64 =
+      "MIIBmTCCAUCgAwIBAgIJASJCAAJVliZXMAoGCCqGSM49BAMCMEUxCzAJBgNVBAYT"
+      + "AkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRn"
+      + "aXRzIFB0eSBMdGQwIBcNMTUwODA1MTY0OTI0WhgPMjA2MzA2MDcxNjQ5MjRaMEUx"
+      + "CzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRl"
+      + "cm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQu"
+      + "CXRfbg9BKnqEs2frC43LSmHR+jNrvs/jC9CiyPr3RzSoL8A0ElifTMEH+TLTFn6W"
+      + "HrZkwwgDR+UFYmwdXRXPoxcwFTATBgsrBgEEAYLlHAIBAQQEAwID2DAKBggqhkjO"
+      + "PQQDAgNHADBEAiBYtS8gXcl3LhvvkVlzYJgpD/tYUHae/Rw3z8lxQSeeXwIgDE2R"
+      + "yWxFfRpgeg0WsLVHu7Ll4oZUkBEuS5RgezrcrRg=";
+  protected static final X509Certificate TRUSTED_CERTIFICATE_EXTENDED_TRANSPORTS =
+      parseCertificateBase64(TRUSTED_CERTIFICATE_EXTENDED_TRANSPORTS_BASE64);
+
+  // Has Bluetooth Radio, Bluetooth Low Energy and NFC
```

Comment:
the comment doesn't make it clear how this is different from TRUSTED_CERTIFICATE_MULTIPLE_TRANSPORTS_BASE64